client/media/install.md

```diff
@@ -0,0 +1,48 @@
+# For beginners
+
+It is recommended to use the [coq-platform](https://github.com/coq/platform?tab=readme-ov-file#installation).
+
+# For developpers
+
+## Opam
+
+[opam](https://opam.ocaml.org/) is the main OCaml package manager. It is the easyiest way to install Coq and Coq packages.
+
+### Installing opam
+The quickest way to install opam is to run this script.
+```
+bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
+```
+Or for Windows users:
+```
+Invoke-Expression "& { $(Invoke-RestMethod https://raw.githubusercontent.com/ocaml/opam/master/shell/install.ps1) }"
+```
+Detailed install instructions can be found [here](https://opam.ocaml.org/doc/Install.html).
```

Comment:
Maybe it should just point to the opam website? Since it might be updated upstream.